features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -486,6 +486,11 @@
         <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.smaenergymeter/${project.version}</bundle>
     </feature>
 
+    <feature name="openhab-binding-somfytahoma" description="Somfy Tahoma Binding" version="${project.version}">
+        <feature>openhab-runtime-base</feature>
+        <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.somfytahoma/${project.version}</bundle>
```

Comment:
please also add
```
         <feature>openhab-transport-http</feature>
```
to make sure that the HTTP client is available

Reply:
Added

Reply:
done